ui/app/AppLayouts/Chat/CommunityColumn.qml

```diff
@@ -41,39 +41,52 @@ Item {
             anchors.topMargin: -4
         }
 
-        StatusIconButton {
+        StatusRoundButton {
             id: optionsBtn
-            icon.name: "dots-icon"
-            iconColor: Style.current.inputColor
+            pressedIconRotation: 45
+            icon.name: "plusSign"
+            size: "medium"
+            width: 36
+            height: 36
             anchors.right: parent.right
             anchors.rightMargin: Style.current.bigPadding
-            anchors.verticalCenter: parent.verticalCenter
-            onClicked: optionsMenu.open()
-        }
+            anchors.top: parent.top
+            anchors.topMargin: 8
 
-        PopupMenu {
-            id: optionsMenu
-            x: optionsBtn.x + optionsBtn.width / 2 - optionsMenu.width / 2
-            y: optionsBtn.height
-
-            Action {
-                enabled: chatsModel.communities.activeCommunity.admin
-                //% "Create channel"
-                text: qsTrId("create-channel")
-                icon.source: "../../img/hash.svg"
-                icon.width: 20
-                icon.height: 20
-                onTriggered: openPopup(createChannelPopup, {communityId: chatsModel.communities.activeCommunity.id})
+            onClicked: {
+                optionsBtn.state = "pressed"
+                let x = optionsBtn.iconX + optionsBtn.icon.width / 2 - optionsMenu.width / 2
+                optionsMenu.popup(x, optionsBtn.icon.height + 14)
             }
 
-            Action {
-                //% "Leave community"
-                text: qsTrId("leave-community")
-                icon.source: "../../img/delete.svg"
-                icon.color: Style.current.red
-                icon.width: 20
-                icon.height: 20
-                onTriggered: chatsModel.communities.leaveCurrentCommunity()
+            PopupMenu {
+                id: optionsMenu
+                x: optionsBtn.x + optionsBtn.width / 2 - optionsMenu.width / 2
+                y: optionsBtn.height
+
+                Action {
+                    enabled: chatsModel.activeCommunity.admin
```

Comment:
```suggestion
                    enabled: chatsModel.communities.activeCommunity.admin
```

Reply:
It was original code.

Reply:
I think something is wrong with your setup then, because the original code is ok: https://github.com/status-im/status-desktop/pull/1982/files#diff-a5e90a78112a0afa0eacacf95aa896eaa0473e26b656f61b561a4991ebb4e036L60
(old line 60)